evap/contributor/tests/test_views.py

```diff
@@ -39,12 +39,21 @@ class TestContributorCourseView(ViewTest):
 
     @classmethod
     def setUpTestData(cls):
-        cls.course = course_with_responsible_and_editor(course_id=TESTING_COURSE_ID)
-
+        course_with_responsible_and_editor(course_id=TESTING_COURSE_ID)
```

Comment:
`create_course_with_responsible_and_editor` would be a better name. could you please change it?